### ISSUE TITLE
cant wake from sleep,uses lots of battery too

### ISSUE THREAD

**Issue by dunk8888** [zkemble/NWatch#2]
how do i wake watch from screen off sleep mode,no buttons seem to wake it up.

**Comment by ZakKemble** [comment]
Any button press should wake the watch. Do all of the buttons work when the watch is on?

**Comment by dunk8888** [comment]
yes all buttons work,this is only pcb desighn,i use it as a little allarm
clock and it seems to use a 300mah battery overnight even when screen is
off,if i leave for more than a few minutes it doesnt like to wake up,i have
to pull battery then plug back in,i love the car game by the way,i love the
hole thing,verry smart indeed.

**Comment by ZakKemble** [comment]
Thanks :D Hmm, did you load the .eep file? Maybe double check for any short circuits too. What's the current draw from the battery like?
